src/scripting_api/aform.js

```diff
@@ -564,7 +564,7 @@ class AForm {
           event.change.length +
           event.selStart -
           event.selEnd;
-        if (finalLen >= 8) {
+        if (finalLen > 8 || event.value[0] === "(") {
```

Comment:
Assuming that `event.value` is indeed a string, perhaps it'd be a little bit clearer to use `startsWith` for the comparison rather than array notation?